<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentAbout</class>
 <widget class="QDialog" name="StudentAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ученик</string>
  </property>
  <widget class="QPushButton" name="pb1_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	
	background-color: qlineargradient(spread:repeat, x1:0.511, y1:0, x2:0.506, y2:1, stop:0 rgba(53, 200, 102, 255), stop:1 rgba(4, 142, 50, 255));
	
    border-style: solid;
    border-color: rgb(147, 147, 147);
    border-width: 1px;
	font: 75 12pt &quot;Segoe Print&quot;;

}
</string>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb1_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	
	background-color: qlineargradient(spread:repeat, x1:0.511, y1:0, x2:0.506, y2:1, stop:0 rgba(53, 200, 102, 255), stop:1 rgba(4, 142, 50, 255));
	
    border-style: solid;
    border-color: rgb(147, 147, 147);
    border-width: 1px;
	font: 75 12pt &quot;Segoe Print&quot;;

}
</string>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMore">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	
	background-color: qlineargradient(spread:repeat, x1:0.511, y1:0, x2:0.506, y2:1, stop:0 rgba(53, 200, 102, 255), stop:1 rgba(4, 142, 50, 255));
	
    border-radius:8px;
    border-style: solid;
    border-color: rgb(147, 147, 147);
    border-width: 1px;
	font: 75 12pt &quot;Segoe Print&quot;;

}
QPushButton:hover{
background-color: rgb(7, 193, 82);
}
QPushButton:pressed{
	background-color: qlineargradient(spread:repeat, x1:0.511, y1:0, x2:0.506, y2:1, stop:0 rgba(53, 200, 102, 255), stop:1 rgba(4, 142, 50, 255));
}
QPushButton:disabled{
	background-color: rgb(116, 163, 135);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Подробнее</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="etSurname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="etPatronymic">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="etName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb1_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	
	background-color: qlineargradient(spread:repeat, x1:0.511, y1:0, x2:0.506, y2:1, stop:0 rgba(53, 200, 102, 255), stop:1 rgba(4, 142, 50, 255));
	
    border-style: solid;
    border-color: rgb(147, 147, 147);
    border-width: 1px;
	font: 75 12pt &quot;Segoe Print&quot;;

}
</string>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="etAdress">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb1_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: white;
	
	background-color: qlineargradient(spread:repeat, x1:0.511, y1:0, x2:0.506, y2:1, stop:0 rgba(53, 200, 102, 255), stop:1 rgba(4, 142, 50, 255));
	
    border-style: solid;
    border-color: rgb(147, 147, 147);
    border-width: 1px;
	font: 75 12pt &quot;Segoe Print&quot;;

}
</string>
   </property>
   <property name="text">
    <string>Адрес</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
